### PR TITLE
Add note about automated decompression

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Adds a note that decompression is automated in versions 2.3 and later.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes https://github.com/timescale/docs/issues/135

### CODE REVIEW

File: timescaledb/how-to-guides/compression/decompress-chunks.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ the chunk first. Inserting data into a compressed chunk is more computationally
 expensive than inserting data into an uncompressed chunk, so decompressing the
 chunk is also a good idea if you need to backfill large amounts of data.
 
+If you are using TimescaleDB 2.3 or later, decompression is automated when you
+ingest older data. For earlier versions of TimescaleDB, you need to perform
+decompression manually.
```
Comment (ryanbooz) [inline timescaledb/how-to-guides/compression/decompress-chunks.md:9]: I'm not 100% sure this is the best way to state this. @svenklemm or @erimatnor. Thoughts?

Comment (Loquacity) [reply timescaledb/how-to-guides/compression/decompress-chunks.md:9]: There's a somewhat related issue in https://github.com/timescale/docs/issues/168 and I've pinged @gayyappan on Slack to ask for assistance. I think we might need to do some more thinking on this section.

Comment (Loquacity) [reply timescaledb/how-to-guides/compression/decompress-chunks.md:9]: It was originally a comment from @jonatas but I might have misunderstood. Is there a better place to put this note? Or should the note read differently?

**Comment by jonatas** [comment]
Looks like it was a miscomprehension and we thought we would have this feature. Closing as the #135 is no longer valid.

**Pull request closed without merging**